changelog.d/gh-5219.fixed

```diff
@@ -0,0 +1 @@
+Scala: Fixed erroneous parsing for `import` in Semgrep patterns.
```

Comment:
This is pretty vague. Maybe something like "Allow metavariables in `import` patterns?"